client/js/lounge.js

```diff
@@ -170,6 +170,10 @@ $(function() {
 		$("#sign-in").remove();
 
 		var id = data.active;
+
+		// Set it right away before triggering a click so that "open" event is not emitted
```

Comment:
Yes, when loading the page, the active channel shouldn't be marked as read to give a chance to the user to catch up on new messages.

Actually, all that chain of PRs (#561, #613, this one, #635) was triggered by trying to not mark channel as read on initial load :D